src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/StringImportSignature.cs

```diff
@@ -41,7 +41,7 @@ public override void AppendMangledName(NameMangler nameMangler, Utf8StringBuilde
 
         public override int CompareToImpl(ISortableNode other, CompilerComparer comparer)
         {
-            return _token.CompareTo(((StringImportSignature)other)._token);
+            return _token.CompareTo(((StringImportSignature)other)._token, comparer);
```

Comment:
The _signatureContext is missing the comparison here. This might probably fail with large version bubble, where 2 strings from 2 different modules have the same token

Reply:
Fixed.